functional/config.rb

```diff
@@ -0,0 +1,58 @@
+# Size of the CoreOS cluster created by Vagrant
```

Comment:
It is much easier to sync modified Vagrantfile with the original one. So let's use config.rb